src/theory/quantifiers_engine.h

```diff
@@ -103,28 +95,33 @@ class QuantifiersEngine {
   std::vector< QuantifiersUtil* > d_util;
   /** vector of modules for quantifiers */
   std::vector< QuantifiersModule* > d_modules;
+  //------------- quantifiers utilities
```

Comment:
Maybe this is a good opportunity to move these private fields after the public ones?